GVFS/GVFS.Hooks/HooksPlatform/GVFSHooksPlatform.Windows.cs

```diff
@@ -16,7 +16,7 @@ public static bool IsElevated()
 
         public static bool IsProcessActive(int processId)
         {
-            return WindowsPlatform.IsProcessActiveImplementation(processId);
+            return WindowsPlatform.IsProcessActiveImplementation(processId, tryGetProcessById: false);

```

Comment:
Why pass in `false` here?

Are the hooks started with the same elevation as their parent git.exe process?

Reply:
If you're able to confirm that could you add a comment describing why we're passing in `false`?